examples/with-tailwind/src/routes/index.tsx

```diff
@@ -0,0 +1,40 @@
+import type { JSX } from 'react'
+import type { TuonoProps } from 'tuono'
+
+interface IndexProps {
+  subtitle: string
+}
+
+export default function IndexPage({
+  data,
```

Comment:
We should not use `data` and `loading` in this template since there is no server side functionality